src/main/java/com/mraof/minestuck/tileentity/redstone/WirelessRedstoneTransmitterTileEntity.java

```diff
@@ -19,8 +21,10 @@
 
 public class WirelessRedstoneTransmitterTileEntity extends TileEntity implements INameable, ITickableTileEntity
 {
-	private BlockPos destBlockPos;
+	private BlockPos offsetPos;
+	private Direction facing;
 	private int tickCycle;
+	private static final int WIRELESS_CONSTANT = 6;
```

Comment:
`WIRELESS_CONSTANT` isn't a very informative name in this context. What is it for?

Reply:
Do you consider my added comment to have resolved this?

Reply:
Alright, with the new comment, I'd suggest renaming the constant to something like `TICK_SPEED`, `TICK_DURATION`, `TICK_FREQUENCY` or `TICK_DELAY`